seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1596,7 +1596,7 @@
         "Here's an example of a radio button:",
         "<code>&#60;label&#62;&#60;input type=\"radio\" name=\"indoor-outdoor\"&#62; Indoor&#60;/label&#62;</code>",
         "<hr>",
-        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>."
+        "Add a pair of radio buttons to your form. One should have the option of <code>indoor</code> and the other should have the option of <code>outdoor</code>. All two should share the <code>name</code> attribute of <code>indoor-outdoor</code>"
```

Comment:
Sure!

@systimotic I am without my computer now. But, I can make these changes later.

Just one question to ponder: I followed the same sentence structure of the next challenge. Can you verify? May be I can change both, or let as is to maintain consistency.

Let me know what do you prefer.